WooCommerce/Classes/ViewRelated/FilterTabBar/FilterTabBar.swift

```diff
@@ -0,0 +1,546 @@
+import UIKit
+
+// NOTE: this file is adapted from WPiOS.
```

Comment:
What do you think of adding a link to the source version?

```suggestion
// NOTE: this file is adapted from WPiOS.
//
// https://github.com/wordpress-mobile/WordPress-iOS/blob/67a5d840eeb9ac027404ad71ed246c645de33687/WordPress/Classes/ViewRelated/System/FilterTabBar.swift
```

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/8942/commits/27adc006d64aaef1db315022feb695c18bb9c640 - the file name is too long so I had to break it into two lines.